src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -95,6 +97,8 @@ def execute(self, emulator=None):
         logging.info('User files are %s', userFiles)
         logging.info('User sandboxes are %s', userTarball)
 
+        scramArch = self.getSingleScramArch(scramArch)
```

Comment:
This may return `None`, right?  Is that handled correctly?  Is it better to fail the job instead?